lib/presenters/contact_presenter.rb

```diff
@@ -0,0 +1,74 @@
+require 'active_support/core_ext/array'
+
+class ContactPresenter
+  attr_reader :contact
+
+  def initialize(contact)
+    @contact = contact
```

Comment:
to avoid all these methods wrapping around the `contact` hash, maybe store `contact` as an openstruct (forgive me, mama) and use `delegate` to access them?